plutarch-test/src/Plutarch/Test/Golden.hs

```diff
@@ -0,0 +1,184 @@
+{-# LANGUAGE ImpredicativeTypes #-}
+
+module Plutarch.Test.Golden (
+  pgoldenSpec,
+  (@|),
+  (@\),
+  (@->),
+
+  -- * Internal
```

Comment:
Internal in what way?

Reply:
Internal to `Plutarch.Test` (and not re-exported, like the list above), but since this module is not exposed, it makes no sense I guess.